v20.1/backup-and-restore.md

```diff
@@ -1,67 +1,72 @@
 ---
 title: Back up and Restore Data
-summary: Learn how to back up and restore a CockroachDB database.
+summary: Learn how to back up and restore a CockroachDB cluster.
 toc: true
 redirect_from:
 - back-up-data.html
 - restore-data.html
 ---
 
-Because CockroachDB is designed with high fault tolerance, backups are primarily needed for disaster recovery (i.e., if your cluster loses a majority of its nodes). Isolated issues (such as small-scale node outages) do not require any intervention. However, as an operational best practice, we recommend taking regular backups of your data.
+Because CockroachDB is designed with high fault tolerance, backups are primarily needed for disaster recovery (i.e., if your cluster loses a majority of its nodes). Isolated issues (such as small-scale node outages) do not require any intervention. However, as an operational best practice, **we recommend taking regular backups of your data**.
 
-Based on your [license type](https://www.cockroachlabs.com/pricing/), CockroachDB offers two methods to back up and restore your cluster's data: Enterprise and Core.
+Based on your [license type](https://www.cockroachlabs.com/pricing/), CockroachDB offers two methods to backup and restore your cluster's data: [Enterprise](#perform-enterprise-backup-and-restore) and [Core](#perform-core-backup-and-restore).
 
 ## Perform Enterprise backup and restore
 
 If you have an [Enterprise license](enterprise-licensing.html), you can use the [`BACKUP`][backup] statement to efficiently back up your cluster's schemas and data to popular cloud services such as AWS S3, Google Cloud Storage, or NFS, and the [`RESTORE`][restore] statement to efficiently restore schema and data as necessary.
 
-### Manual full backups
+{{site.data.alerts.callout_success}}
+We recommend [automating daily backups of your cluster](#automated-full-and-incremental-backups). To automate backups, you must have a client send the `BACKUP` statement to the cluster. Once the backup is complete, your client will receive a `BACKUP` response.
+{{site.data.alerts.end}}
+
+### Full backups
 
 In most cases, it's recommended to use the [`BACKUP`][backup] command to take full nightly backups of your cluster:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO '<full_backup_location>';
+> BACKUP TO '<backup_location>';
 ~~~
 
 If it's ever necessary, you can then use the [`RESTORE`][restore] command to restore your cluster:
```

Comment:
I think it'd be helpful to point out that you can also restore individual databases or tables from the cluster backup. IMO that highlights to the user that the full cluster backup is really all they need.